modules/phase_field/doc/content/source/kernels/ADCHSplitChemicalPotential.md

```diff
@@ -0,0 +1,16 @@
+# ADCHSplitChemicalPotential
+
+!syntax description /Kernels/ADCHSplitChemicalPotential
+
+Uses automatic differentiation to solve for chemical potential in a weak sense (mu-mu_prop=0).  Can
```

Comment:
This is a parallel class to `ADSplitCHParsed` :-/

Reply:
`ADSplitCHParsed` takes the concentration variable and a material free energy.  It returns the material property holding the derivative of the free energy with respect to the concentration:

`_dFdc(getADMaterialProperty<Real>(derivativePropertyNameFirst(_f_name, _var.name())))`
...
`return _dFdc[_qp];`

Our application needs to accept a chemical potential variable and a material chemical potential.  This allows the material chemical potential to be supplied by an external solver (or other means) without ever explicitly defining the free energy.  

What we're proposing is an AD version of `CHSplitChemicalPotential`.  I don't think this provides a parallel calculation to `SplitCHParsed`.

Reply:
Ah, ok, that seems reasonable. I'll go ahead and approve this for merging. At some point it would be nice to link the docs of the AD and non-AD versions and to add a `use_automatic_differentiation` option to the phase field actions.

Reply:
Ugh, it even stated that in the description of the PR...

Reply:
Sounds good, thanks again!  I think having linked (or even combined) documentation for objects that have both standard and AD versions is a good idea.